src/components/grid-aware/ImageHeader/ImageHeader.module.css

```diff
@@ -0,0 +1,139 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
+  }
+}
+
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+}
+
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+    padding-top: 30px;
+  }
+}
+
+
+/* Main grid areas */
+
+.gridAreaText {
+  grid-column: 2 / span 6;
+  grid-row: 1;
+}
+
+.gridAreaImage1 {
+  grid-column: 9 / span 4;
+  grid-row: 1;
+}
+
+.gridAreaImage2 {
+  grid-column: 10 / span 3;
+  grid-row: 1;
+}
+
+@media (--tablet-and-down) {
+  .gridAreaImage2 {
+    text-align: right;
+  }
+}
+
+
+/* Left grid area */
+
+.title {
+  font: var(--font-headline);
+  margin-top: 90px;
```

Comment:
Rather than defining this on the `.title` element, I think it would make more sense to define this on the `. gridAreaText` element, since it makes more sense for `.gridAreaText` to describe the spacing on the "outside" of all of the elements.

In addition, you probably also want to set the bottom margin on the `.gridAreaText` element, since without that, the bottom edge of the overall `.bleedWrapper`/`<ImageHeader>` component is exactly flush with the CTA button, which means that if we placed another section just below this, there will be no spacing at all.

You can see this visually in the following screenshot, where the blue area (`.bleedWrapper`) ends immediately after the Apply button:

<img width="1298" alt="Screen Shot 2020-11-11 at 10 05 22 PM" src="https://user-images.githubusercontent.com/1002748/98902252-2cb31800-246a-11eb-9467-f0c37c30da1f.png">